Jenkinsfile

```diff
@@ -95,7 +95,7 @@ def deploy(deploy_environment) {
         sh("eval \$(aws ecr get-login --no-include-email)")
         def appImage = docker.build(
           "govwifi/authorisation-api:${deploy_environment}",
-          "--build-arg BUNDLE_INSTALL_CMD='bundle install --without test' ."
+          "--build-arg BUNDLE_INSTALL_CMD='bundle install --without test' --build-arg RACK_ENV='${deploy_environment}' ."
```

Comment:
What made you decide to use a build arg and not an `ENV`?  Feels more like a runtime variable, instead of a compile time variable.

Reply:
We build separate images for both staging and production so I thought it made sense to bake it in at that the time of build rather than relying on yet more config via terraform